Source/Scene/Camera.js

```diff
@@ -3230,9 +3241,39 @@ define([
         Matrix4.clone(camera._transform, result.transform);
         result._transformChanged = true;
 
+        if (defined(camera._boundingObject)) {
+            result.boundingObject = camera._boundingObject.clone(result.boundingObject);
+        }
+
         return result;
     };
 
+    var options = {
+        destination : new Cartesian3(),
+        orientation : new HeadingPitchRoll()
+    };
+
+    /**
+     * @private
+     */
```

Comment:
This block comment is describing and should be above `Camera.prototype._limitPosition`, not `var scratchClampedValue`